cmd/helm/install.go

```diff
@@ -285,12 +286,32 @@ func (v *values) Set(data string) error {
 	return nil
 }
 
+func typedVal(val string) interface{} {
+	if val == "true" {
```

Comment:
My feeling is that we should treat 0 and 1 as integers. But we probably should `string.ToLower()` val before comparing it.